test/fixtures/line/labelShadowColors.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  tolerance: 0.0015,
```

Comment:
I don't think any of the changes in this fixture are actually needed (at least my tests work fine when removed)

Reply:
it was weird because I didn't change anything as well but when tested in FF, the tester found some differences on the label text  on the top of the chart. Let me retry

Reply:
Here the failed test in FF:

![image](https://user-images.githubusercontent.com/11741250/168226060-c9724609-2f2f-4bb1-a337-ce57621fef6f.png)

Maybe you have seen the some behavior in other cases.

Reply:
@kurkle interesting thing.... I have removed the tolerance and commited and it works in CI. But it's not working to me locally.
I see a difference: I'm using FF 100 and the CI is using 99....

Reply:
I can manage that locally (annoying but no problem!)

Reply:
the added tolerance is fine, ci will probably also fail with ff100

Reply:
OK, wait for next push, I'll reapply